ansible/roles/postfix/vars/main/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+postfix_mailname: "pydis.wtf"
+
+postfix_bind_user: "uid=postfix,cn=users,cn=accounts,dc=box,dc=pydis,dc=wtf"
+postfix_bind_password: "{{ vault_postfix_bind_password }}"
+postfix_bind_server: "ldaps://ldap01.box.pydis.wtf:636"
```

Comment:
I have yet to add the validation stage but yes there is an LDAP flag that can be added. I wanted to enroll the host first as it would ensure the up-to-date CA is present on the host from what I understand.